tests/core/test_selector.py

```diff
@@ -392,6 +392,7 @@ def test_model_selection_normalized(mocker: MockerFixture, make_snapshot):
     [
         (["git:main"], {'"test_model_a"', '"test_model_c"'}),
         (["git:main & +*model_c"], {'"test_model_c"'}),
+        (["git:main+"], {'"test_model_a"', '"test_model_c"', '"test_model_d"'}),
```

Comment:
Do we need a test for the leading `+` too?

Reply:
Good call that actually caught a bug

Reply:
Addressed: https://github.com/TobikoData/sqlmesh/pull/2905/commits/a5a208158fb7dd116d6d53ed3be6fa1db20ca3f7